nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -84,6 +73,7 @@ def run_conversion(
         chunk_data: bool = True,
         module_name: Optional[str] = None,
         module_description: Optional[str] = None,
+        compression: str = "gzip",
```

Comment:
```suggestion
        compression: str = "gzip",
        compression_options: Optional[dict] = None,
```
Might as well include compression options as well.

Reply:
yes, thanks for that. Indeed we can use compression options to input to `H5DataIO` https://github.com/catalystneuro/nwb-conversion-tools/pull/372/commits/bb68d9c4c651252350560d2b0104e09ad1a8adc1

Reply:
Of course, it will need aliasing in the actual `H5DataIO` call since there it is called ` compression_opts` (less descriptive imo)